xclim/indices/stats.py

```diff
@@ -90,8 +91,8 @@ def fit(
         Name of the univariate distribution, such as beta, expon, genextreme, gamma, gumbel_r, lognorm, norm
         (see :py:mod:scipy.stats for full list). If the PWM method is used, only the following distributions are
         currently supported: 'expon', 'gamma', 'genextreme', 'genpareto', 'gumbel_r', 'pearson3', 'weibull_min'.
-    method : {"ML", "PWM"}
-        Fitting method, either maximum likelihood (ML) or probability weighted moments (PWM), also called L-Moments.
+    method : {"ML", "PWM", "APP"}
+        Fitting method, either maximum likelihood (ML), probability weighted moments (PWM), also called L-Moments, or approximate method (APP)
```

Comment:
I'm a bit confused by the APP method. It essentially returns the value of _fit_start, which is just a rough estimate of initial values.

Reply:
What I mean is that I'm not sure it's a good idea to expose those initial values as a legitimate fit methods in the public interface.

Reply:
This was introduced in a previous PR for the SPI/SPEI using only the gamma distribution and we were comparing with the library `monocongo/climate_indices` (used by `drought.gov`) which only uses the approximate method as in  `_fit_start`.  Is the issue here that we label this as a "gamma fit" even though only approximate methods are used? Or is the concern about the usefulness of such results? When I was developing this, I saw some rare instances where the approximate method seem more robust than the full method using ``scipy``, but I'm not sure I noted these values: I can try to find them back.

Reply:
Yes, my concern is that we're exposing an approximate fitting method, appropriate for this case, inside a generic function.

Reply:
Ok, I see. Do you think that calling `_fit_start`  directly within `standardardized_precipitation_index` and `standardardized_precipitation_evapotranspiration_index` if the method "APP" is given as input could be an acceptable alternative?

Reply:
Hum, this is going to add complexity. For me, the best option would be to remove the APP method from the public interface, and make sure the scipy fit function works well, but that might prove tricky or require upstream changes. I'll keep thinking about this.